web/src/components/trace/TracePreview.tsx

```diff
@@ -20,15 +20,20 @@ import ScoresTable from "@/src/components/table/use-cases/scores";
 import { ScoresPreview } from "@/src/components/trace/ScoresPreview";
 import { AnnotateDrawer } from "@/src/features/scores/components/AnnotateDrawer";
 import useLocalStorage from "@/src/components/useLocalStorage";
+import { CommentDrawerButton } from "@/src/features/comments/CommentDrawerButton";
 
 export const TracePreview = ({
   trace,
   observations,
   scores,
+  showComments,
```

Comment:
why is showcomments necessary?

Reply:
Because we don't show comments on the trace timeline view and this currently controls that

Reply:
ok, lets remove